### PR TITLE
Io 45 add session ids

### PR DESCRIPTION
**Pull request by nilsmart**
We are adding new messages for getting analytics around player session and video sessions to pull metrics across adtech and video tech.

### CODE REVIEW

File: js/utils/constants.js [hunk 1]
```diff
@@ -1101,6 +1101,39 @@
...
+       *     <li>The ID (session id) of the player.</li>
+       *   </ul>
+       *
+       *
+       * @event OO.EVENTS#SESSION_ID_SET
+       */
+      SESSION_ID_SET: "sessionIdSet",
+
+      /**
+       * An attempt has been made to set content session id. This will get sent out when new asset is loaded.
```
Comment (dchauOoyala) [inline js/utils/constants.js]: - The comments in these three messages seem to all follow: "An attempt has been made to set..."
We have `SESSION_ID_SET` above; should we be consistent here and also have `CONTENT_SESSION_ID_SET` and `AD_BREAK_SESSION_ID_SET`. 

- As a follow-up question. The comment suggests that an "attempt" was made, but the message itself suggests that it has been already been set.

**Comment by dchauOoyala** [comment]
Is there a corresponding pull request to the mjolnir repo for these message additions?

**Comment by pilievOoyala** [comment]
There will be later. I just remembered how to let Nilesh test what he wants without this being merged yet. You can hold off on merging.

File: js/utils/constants.js (hunk 1 above)
Comment (nilsmart) [inline js/utils/constants.js]: I will update these names.

**Comment by aldigun** [comment]
We will be closing this PR for now.

**Pull request closed without merging**